.trivyignore

```diff
@@ -0,0 +1,6 @@
+## DOCKER
+
+# ADD is used for getting file from the internet
+AVD-DS-0005
+# Healthcheck already exists
+AVD-DS-0026
```

Comment:
I don't think is is true - we have a healthcheck in the docker-compose and in ECS config, but we don't have one in our dockerfile itself

Reply:
Will update